checker/bin-devel/check-jdk-consistency.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set +o xtrace  # Turn off xtrace to avoid verbose output
```

Comment:
How about doing something like described here: https://stackoverflow.com/questions/14564746/in-bash-how-to-get-the-current-status-of-set-x/50668339#50668339
to only toggle, here and at the end, if something needs to happen?
Otherwise, we might leave with xtrace turned on, even though it was off before.